lib/ui/workoutPlayer/workout_player.dart

```diff
@@ -0,0 +1,234 @@
+      Exercise(5, "Exercise 5", "Exercise5 Description", dummyThumbnailUrl, dummyVideoUrl, 90),
+      Exercise(6, "Exercise 6", "Exercise6 Description", dummyThumbnailUrl, dummyVideoUrl, 120),
+      Exercise(7, "Exercise 7", "Exercise7 Description", dummyThumbnailUrl, dummyVideoUrl, 191),
+      Exercise(8, "Exercise 8", "Exercise8 Description", dummyThumbnailUrl, dummyVideoUrl, 1000),
+    ];
+  }
+
+  void setCurExercise(int index) {
+    setState(() {
+      curExerciseIndex = index;
+    });
+  }
+
+  void playNextExercise() {
+    if (curExerciseIndex + 1 < exercises.length) {
+      this.setCurExercise(curExerciseIndex + 1);
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    var scaffold = Scaffold(
+      body: Row(
+        children: [
+          Expanded(
+            flex: 3,
+            child: Column(
+              children: [
+                videoPlayerFunction(context, exercises[curExerciseIndex].videoSrc, this.playNextExercise),
+                videoDescription(),
+              ]
+            ),
+          ),
+          Expanded(
+            flex: 1,
+            child: Container(
+              color: Colors.orange,
+              child: Column(
+                children: [
+                  workoutTitle(),
+                  Expanded(
+                    child: exerciseList(context, exercises, setCurExercise, _itemScrollController, curExerciseIndex),
+                  )
+                ],
+              )
+            )
+          )
+        ]
+      ),
+    );
+    return scaffold;
+  }
+}
+
+Widget videoPlayerFunction(BuildContext context, String videoSrcUrl, Function handleVideoCompleted)  {
+  return Stack(
+    children: <Widget> [
+      Container(
+        color: Colors.black,
+        alignment: Alignment.center,
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height * 0.75,
+        child: AspectRatio(
+          aspectRatio: 16/9,
+          child: VideoPlayer(videoSrcUrl, UniqueKey(), handleVideoCompleted)
+        ),
+      ),
+      countDownTimer(context)
+    ]
+  );
+}
+
+Widget countDownTimer(BuildContext context) {
+  return Positioned(
+    top: 0,
+    left: MediaQuery.of(context).size.width / 30,
+    child: CircularCountDownTimer(
+      duration: 30,
+      initialDuration: 0,
+      width: MediaQuery.of(context).size.width / 30,
+      height: 200,
+      ringColor: Colors.grey,
+      fillColor: Colors.white,
+      strokeWidth: 3.0,
+      strokeCap: StrokeCap.round,
+      textStyle: const TextStyle(
+        fontSize: 20.0,
+        color: Colors.white,
+        fontWeight: FontWeight.bold
+      ),
+      textFormat: CountdownTextFormat.S,
+      isReverse: true,
+      isTimerTextShown: true,
+      autoStart: true,
+    )
+  );
+}
+
+Widget videoDescription() {
+  return const Text('This is the workout description', textAlign: TextAlign.left,);
```

Comment:
Make sure to update these as well!

Reply:
This is the "exercise" description right?

Reply:
I believe this is the description of the "workout"? Don't think we have a field for "workout" description in db, but this is what I remembered it to be?
Currently the exercise description shows up when user clicks on the dropdown arrow on the right of the exercise thumbnail and name. Also put a picture in the PR description showing where the "exercise" descriptions are

Reply:
I don't think we ended up including it in the DB since that's additional functionality for the user (ie to add in when creating/editing a workout)- maybe we can remove this for now and create a ticket to discuss if this is something we want?

Reply:
RESOLVED -> Adding as P2 feature